package.json

```diff
@@ -39,34 +39,47 @@
     "lint:report": "eslint . --output-file ./eslint_report.json --format json  --ext .ts,.js,.tsx | true"
   },
   "dependencies": {
-    "react-native-modal": "11.5.6"
+    "react-native-modal": "13.0.1"
   },
   "peerDependencies": {
-    "react": ">=16.8.3",
-    "react-native": ">=0.59.0"
+    "react": "*",
+    "react-native": ">=0.65.0"
   },
   "devDependencies": {
-    "@babel/preset-env": "7.12.1",
-    "@babel/preset-react": "7.12.1",
-    "@babel/preset-typescript": "7.12.1",
+    "@babel/core": "7.13.15",
```

Comment:
What is this change for? adding a lot of dependencies.

Reply:
it's for unit testing, 

before adding:
https://github.com/oolio-group/rn-use-modal/actions/runs/7028800357/job/19125360359